### PR TITLE
Add check to addSubscriberButtons and fix item count

### PR DESCRIPTION
**Pull request by candela97**
The subscribe feature only works if user is logged in, so add some checks to prevent uncaught errors on for example https://steamcommunity.com/workshop/browse/?appid=221380

Also fixed total item count, see https://github.com/tfedor/AugmentedSteam/pull/722#issuecomment-582896957

### CODE REVIEW

**Comment by MxtOUT** [comment]
I don't see any errors when I visit this page

**Comment by candela97** [comment]
I tested again and found the rightSectionTopTitle section won't appear if not logged in. The subscribe function won't work if the user isn't logged in anyway, so I'll add a check for that instead.

**Comment by candela97** [comment]
I also found the method to count the total number of workshop items isn't quite reliable.
```          
let total = parseInt(
    pagingInfo.textContent
        .replace(/\d+-\d+/g, "")
        .match(/\d+/g)[0]);
```
1. `replace(/\d+-\d+/g, "")` can be modified by the localization process, so it won't work for languages like Simplified Chinese (contains additional spaces) and German (uses a different type of hyphen)
2. `match(/\d+/g)[0]` won't work properly if the number of items exceeds 1,000

I think it's better if we remove the comma, then find the largest number.

File: js/content/community.js [hunk 1]
```diff
@@ -3841,10 +3841,7 @@ let WorkshopBrowseClass = (function(){
 
         document.querySelector("#es_subscriber_container").addEventListener("click", e => {
             let method = e.target.closest(".es_subscriber").dataset.method;
-            let total = parseInt(
-                pagingInfo.textContent
-                    .replace(/\d+-\d+/g, "")
-                    .match(/\d+/g)[0]);
+            let total = Math.max(...pagingInfo.textContent.replace(",", "").match(/\d+/g));
```
Comment (Revadike) [inline js/content/community.js]:
```suggestion
            let total = Math.max(...pagingInfo.textContent.replace(/,/g, "").match(/\d+/g));
```

**Pull request merged**